system_setup/server/controllers/configure.py

```diff
@@ -68,13 +72,135 @@ async def configure(self, *, context):
 
             self.app.update_state(ApplicationState.POST_RUNNING)
 
+            variant = self.app.variant
+            if variant == "wsl_setup":
+                wsl_identity = self.model.identity
+                run_command(["/usr/sbin/useradd", "-m", "-s", "/bin/bash",
+                             "-p", wsl_identity.password,
+                             wsl_identity.username])
+                run_command(["/usr/sbin/usermod", "-a",
+                             "-c", wsl_identity.realname,
+                             "-G", self.get_userandgroups(),
+                             wsl_identity.username])
+
+                wslconf_base = self.model.wslconfbase
+                run_command(["/usr/bin/ubuntuwsl", "update",
```

Comment:
There's a lot of repetition here.  A utility function taking the args  "WSL.automount.root" and wslconf_base.custom_path could really help shorten this.  About 22 usages of this same construct.

Reply:
This is not what I suggested. I suggested merging TODO DE101 and DE102, only 104 can stay aside (for dry-run mode). However, 102 is about changing ubuntu-wsl to write directly on system wsl.conf